.storybook/themes/news.scss

```diff
@@ -633,6 +633,46 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					box-shadow: (
+						0 4px 8px 0 rgba(0, 0, 0, 0.2),
+						0 6px 20px 0 rgba(0, 0, 0, 0.19),
+					),
+					border-radius: 0 5px 5px 0,
+					display: flex,
+					left: 0,
+					padding: 9px 0,
+					position: fixed,
+					top: 200px,
+					width: 44px,
+					components: (
+						button-medium: (
+							background: transparent,
+							border-width: 0,
+							padding: 7px 10px,
+							text-align: center,
+						),
+						icon: (
+							height: var(--global-spacing-5),
```

Comment:
height shouldn't be set in styles. it should be set with props for the theme component